packages/amplify-graphql-model-transformer/src/types.ts

```diff
@@ -0,0 +1,9 @@
+import { SyncConfig } from '@aws-amplify/graphql-transformer-core';
+
+export type ModelTransformerOptions = {
+  EnableDeletionProtection?: boolean;
+  SyncConfig?: SyncConfig;
+};
+
+export const MYSQL_DB_TYPE = 'MySQL';
+export const DDB_DB_TYPE = 'DDB';
```

Comment:
Would love to see these moved to the GQL core as well.

Reply:
Moved locally and about to commit